src/cascadia/TerminalControl/TSFInputControl.cpp

```diff
@@ -0,0 +1,379 @@
+#include "pch.h"
+#include "TSFInputControl.h"
+#include "TSFInputControl.g.cpp"
+
+using namespace winrt::Windows::Foundation;
+using namespace winrt::Windows::Graphics::Display;
+using namespace winrt::Windows::UI::Core;
+using namespace winrt::Windows::UI::Text;
+using namespace winrt::Windows::UI::Text::Core;
+using namespace winrt::Windows::UI::Xaml;
+
+// hack to account for offset caused by tabs

```

Comment:
Correctable? Fixable? leave a TODO?

Reply:
Using:
`this->TransformToVisual(nullptr).TransformPoint(winrt::Windows::Foundation::Point(0, 0));`
I'm able to get the offset within the window caused by Tabs and margins